streamx-flink/streamx-flink-shims/streamx-flink-shims-base/src/main/scala/com/streamxhub/streamx/flink/core/SqlCommandParser.scala

```diff
@@ -311,178 +317,205 @@ object SqlCommand extends enumeratum.Enum[SqlCommand] {
   case object USE_MODULES extends SqlCommand(
     "use modules",
-    "(USE\\s+MODULES\\s+.*)"
+    "(USE\\s+MODULES\\s+.+)"
   )
 
   /**
    * USE [catalog_name.]database_name
    */
-  case object USE extends SqlCommand(
-    "use",
-    "USE\\s+.*"
+  case object USE_DATABASE extends SqlCommand(
+    "use database",
+    "(USE\\s+(?!(CATALOG|MODULES)).+)"
   )
 
-  //----SHOW Statements
+  //----SHOW Statements--------------------------------------------------------------------------------------------------------------------------------
 
   /**
    * SHOW CATALOGS
    */
   case object SHOW_CATALOGS extends SqlCommand(
     "show catalogs",
-    "SHOW\\s+CATALOGS",
-    Converters.NO_OPERANDS
+    "(SHOW\\s+CATALOGS\\s*)"
   )
 
   /**
    * SHOW CURRENT CATALOG
    */
   case object SHOW_CURRENT_CATALOG extends SqlCommand(
     "show current catalog",
-    "SHOW\\s+CURRENT\\s+CATALOG",
-    Converters.NO_OPERANDS
+    "(SHOW\\s+CURRENT\\s+CATALOG\\s*)"
   )
 
   /**
    * SHOW DATABASES
    */
   case object SHOW_DATABASES extends SqlCommand(
     "show databases",
-    "SHOW\\s+DATABASES",
-    Converters.NO_OPERANDS
+    "(SHOW\\s+DATABASES\\s*)"
   )
 
   /**
    * SHOW CURRENT DATABASE
    */
   case object SHOW_CURRENT_DATABASE extends SqlCommand(
     "show current database",
-    "SHOW\\s+CURRENT\\s+DATABASE",
-    Converters.NO_OPERANDS
+    "(SHOW\\s+CURRENT\\s+DATABASE\\s*)"
   )
 
   /**
-   * SHOW TABLES
+   * SHOW TABLES,support from flink-1.13.x<br>
+   * SHOW TABLES [ ( FROM | IN ) [catalog_name.]database_name ] [ [NOT] LIKE `<sql_like_pattern`> ],support from flink-1.15.x
    */
   case object SHOW_TABLES extends SqlCommand(
     "show tables",
-    "SHOW\\s+TABLES",
-    Converters.NO_OPERANDS
+    "(SHOW\\s+TABLES.*)"
+  )
+
+  /**
+   * SHOW CREATE TABLE，flink-1.14.x support.
+   */
+  case object SHOW_CREATE_TABLE extends SqlCommand(
+    "show create table",
+    "(SHOW\\s+CREATE\\s+TABLE\\s+.+)"
+  )
+
+  /**
+   * SHOW COLUMNS ( FROM | IN ) [`[`catalog_name.]database.]`<table_name>` [ [NOT] LIKE `<sql_like_pattern>`]，flink-1.15.x support.
+   */
+  case object SHOW_COLUMNS extends SqlCommand(
+    "show columns",
+    "(SHOW\\s+COLUMNS\\s+.+)"
   )
 
   /**
    * SHOW VIEWS
    */
   case object SHOW_VIEWS extends SqlCommand(
     "show views",
-    "SHOW\\s+VIEWS",
-    Converters.NO_OPERANDS
+    "(SHOW\\s+VIEWS\\s*)"
+  )
+
+  /**
+   * SHOW CREATE VIEW
+   */
+  case object SHOW_CREATE_VIEW extends SqlCommand(
+    "show create view",
+    "(SHOW\\s+CREATE\\s+VIEW\\s+.+)"
   )
 
   /**
    * SHOW [USER] FUNCTIONS
    */
   case object SHOW_FUNCTIONS extends SqlCommand(
     "show functions",
-    "SHOW\\s+(USER\\s+|)FUNCTIONS",
-    Converters.NO_OPERANDS
+    "(SHOW\\s+(USER\\s+|)FUNCTIONS\\s*)"
   )
 
   /**
    * SHOW [FULL] MODULES
    */
   case object SHOW_MODULES extends SqlCommand(
     "show modules",
-    "SHOW\\s+(FULL\\s+|)MODULES",
-    Converters.NO_OPERANDS
+    "(SHOW\\s+(FULL\\s+|)MODULES\\s*)"
```

Comment:
`\\s*` I don't think it's necessary

Reply:
If I add some Spaces to the end of the semicolon, I find that we didn't get rid of those Spaces when we parses, and it's going to make a mistake when we check, so I add Spaces to the end of all the simple syntax.